.gitignore

```diff
@@ -7,6 +7,7 @@ backend/app/__pycache__
 backend/app/controllers/__pycache__
 backend/__pycache__
 backend/app/__pycache__
+backend/ml_model/model_with_score.pkl
```

Comment:
Why are we adding this in gitignore? Dont we need this?

Reply:
We're adding this to the .gitignore because it's a dynamically generated file that changes frequently. Every time we pull, the file might have been changed, and we'd need to commit it first